templates/none/routes/index.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+
+router.get('/', function (req, res, next) {
+  res.sendFile('index.html', {
```

Comment:
This is just a general comment for discussion, not a request to change : does using `res.sendFile` make sense here, so should it just be let up to `express.static`?